src/prog_translator.h

```diff
@@ -27,14 +36,7 @@ class ExchangeNodeGroup;
 class ProgTranslator {
  public:
   /// @brief struct to hold all problem instance state
-  struct Context {
-    std::vector<double> obj_coeffs;
-    std::vector<double> row_ubs;
-    std::vector<double> row_lbs;
-    std::vector<double> col_ubs;
-    std::vector<double> col_lbs;
-    CoinPackedMatrix m;
-  };
+  typedef ProgTranslatorContext Context;
```

Comment:
For what it is worth, it is this typedef that is incorrect and should be a struct whose constructor throws an error.